src/app/nav.tsx

```diff
@@ -0,0 +1,111 @@
+"use client";
+import styles from "./nav.module.css";
+import React from "react";
+import { useState } from "react";
+
+function Navbar() {
+  const navItems = ["Pokemon Lab", "Battle", "Poke Mart", "My Pokedex"];
+  const [isNavExpanded, setIsNavExpanded] = useState(false);
+  return (
+    <>
+      <div className={styles.navContainer}>
+        <div className={styles.icons}>
+          <img
+            src="/images/hamburger.png"
+            className={styles.hamburgerIcon}
+            onClick={() => {
+              setIsNavExpanded(!isNavExpanded);
+            }}
+            style={{ display: isNavExpanded ? "none" : "block" }}
+          ></img>
+          <div
+            className={styles.closeIconContainer}
+            style={{ position: isNavExpanded ? "static" : "absolute" }}
```

Comment:
It is better to toggle a className that applies those properties instead of toggling properties directly.